haystack_experimental/core/__init__.py

```diff
@@ -2,6 +2,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+from .pipeline.base import ComponentPriority, PipelineBase
 from .super_component import SuperComponent
 
-_all_ = ["SuperComponent"]
+_all_ = ["ComponentPriority", "PipelineBase", "SuperComponent"]
```

Comment:
We don't need to expose ComponentPriority and PipelineBase here or do we? We don't expect users to work with those

Reply:
this was just copy-paste verbatim from main haystack to have a fully functional pipeline - I will adjust